### PR TITLE
small updates to vsphere readme

### PR DESCRIPTION
**Pull request by christianh814**
Small clarifying updates.

### CODE REVIEW

File: docs/vsphere-prerequisites.md [hunk 1]
```diff
@@ -1,27 +1,34 @@
 # vSphere pre-requisites
+
+In order to use the Windows MachineConfig Operator to successfully create a Windows node, you will first need to create a a Windows "golden image" VM. These steps are  things that must be done, and things the Operator expects to be there when the Windows golden image VM is cloned.
```
Comment (aravindhp) [inline docs/vsphere-prerequisites.md:3]: In your 2nd sentence, I suggest saying something like `These steps are an overview of the required configuration of the golden image.` I am sure @codyhoag can wordsmith it better than both of us :-)

**Review by aravindhp — CHANGES_REQUESTED** [review]
Thanks for working on this, @christianh814. Mostly LGTM. Please stick to 120 columns for the doc. If you are using a link then move the link and it's text to the next line.

File: docs/vsphere-prerequisites.md (hunk 1 above)
Comment (christianh814) [reply docs/vsphere-prerequisites.md]: Updated!

**Review by aravindhp — APPROVED** [review]
/approve

@codyhoag please review and lgtm

File: docs/vsphere-prerequisites.md [hunk 2]
```diff
@@ -1,31 +1,66 @@
 # vSphere pre-requisites
+
+In order to use the Windows MachineConfig Operator to successfully create
+a Windows node, you will first need to create a a Windows "golden image"
```
Comment (codyhoag) [inline docs/vsphere-prerequisites.md:4]:
```suggestion
a Windows node, you will first need to create a Windows "golden image"
```

File: docs/vsphere-prerequisites.md [hunk 3]
```diff
@@ -1,31 +1,66 @@
...
+Administrator's home directory stays intact with the SSH public key.
+
+  Here is an example command to `sysprep`
   ```
   C:\Windows\System32\Sysprep\sysprep.exe /generalize /oobe /shutdown /unattend:<path_to_unattend.xml>`
   ```
-  An example [unattend.xml](unattend.xml) is provided however the fields in the example require customization and it
-  should not be used directly.
+  You can find an example [unattend.xml](unattend.xml) in this repo. It
+  is provided as an examplen and it **should not** be used directly.
```
Comment (codyhoag) [inline docs/vsphere-prerequisites.md:46]:
```suggestion
  is provided as an example and it **should not** be used directly.
```

**Review by codyhoag — APPROVED** [review]
Just a couple typo suggestions. All else LGTM!

**Comment by aravindhp** [comment]
@christianh814 please squash merge and we can get this merged.

**Comment by christianh814** [comment]
@aravindhp Squashed PTAL

**Comment by aravindhp** [comment]
@christianh814 I have already approved. @codyhoag please lgtm.

**Comment by aravindhp** [comment]
/cherry-pick community-4.7

to check if the OKD e2e is passing

**Pull request merged**